src/CONST.ts

```diff
@@ -4647,7 +4671,12 @@ const CONST = {
             '\n' +
             "We'll send a request to each person so they can pay you back. Let me know if you have any questions!",
     },
-
+    ONBOARDING_ACCOUNTING_MAPPING: {
+        [connectNames.QBO]: 'QuickBooks Online',
+        [connectNames.XERO]: 'Xero',
+        [connectNames.NETSUITE]: 'NetSuite',
+        [connectNames.SAGE_INTACCT]: 'Sage Intacct',
+    },
```

Comment:
```suggestion
```

We have translation key for that `workspace.accounting.connectionName`

Reply:
As below, we don't translate the task.

Reply:
RIght but those connection names are not translated either way. It's just better to avoid maintaining another map that holds an existing data